### PR TITLE
[BasicExchangeWrapper] add BasicExchangeWrapper

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
for https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/577

File: octobot_trading/api/exchange.py [hunk 1]
```diff
@@ -18,32 +18,52 @@
...
+    :param exchange_lib: octobot_trading.enums.ExchangeWrapperLibs to use
+    :return: the list of time frames
+    """
+    try:
+        # first try in available exchanges
+        for exchange_configuration in get_exchange_configurations_from_exchange_name(exchange_name).values():
+            return exchange_configuration.exchange_manager.client_time_frames
+    except KeyError:
+        async with exchanges.temporary_exchange_wrapper(exchange_name, exchange_lib) as wrapper:
+            return list(await wrapper.get_available_time_frames())
```
Comment (Herklos) [inline octobot_trading/api/exchange.py:58]: :+1:

File: octobot_trading/exchanges/basic_exchange_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import contextlib
+import ccxt.async_support
+import ccxt
+
+import octobot_trading.enums as enums
+
+
+class BasicExchangeWrapper:
```
Comment (Herklos) [inline octobot_trading/exchanges/basic_exchange_wrapper.py:23]: Nice implementation :+1:

**Review by Herklos — COMMENTED** [review]
I love the implementation :100:

**Review by Herklos — CHANGES_REQUESTED** [review]
You forgot to add the new pxd files to setup.py compile list

**Review by Herklos — APPROVED** [review]

**Pull request merged**